### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by simondeziel**

### CODE REVIEW

**Comment by simondeziel** [comment]
@masnax it seems the raft part is build with liblz4 support but the corresponding .so doesn't make it to the resulting snap as it's not primed:

```
root@c1:/snap/microcloud/current# ll lib/
total 2309
lrwxrwxrwx 1 root root      18 Feb  2 18:14 libdqlite.so -> libdqlite.so.0.0.1*
lrwxrwxrwx 1 root root      18 Feb  2 18:14 libdqlite.so.0 -> libdqlite.so.0.0.1*
-rwxr-xr-x 1 root root 1251240 Feb  2 18:14 libdqlite.so.0.0.1*
lrwxrwxrwx 1 root root      16 Feb  2 18:14 libraft.so -> libraft.so.3.0.0*
lrwxrwxrwx 1 root root      16 Feb  2 18:14 libraft.so.3 -> libraft.so.3.0.0*
-rwxr-xr-x 1 root root 1113976 Feb  2 18:14 libraft.so.3.0.0*
drwxr-xr-x 2 root root       0 Feb  2 18:14 x86_64-linux-gnu/
root@c1:/snap/microcloud/current# ll lib/x86_64-linux-gnu/
total 194
lrwxrwxrwx 1 root root     14 Jan 15  2022 libuv.so.1 -> libuv.so.1.0.0
-rw-r--r-- 1 root root 198744 Jan 15  2022 libuv.so.1.0.0
```

Should I prime it? Or maybe disable lz4 support?

**Comment by masnax** [comment]
Well MicroCloud's database is very basic (it's only a record of cluster members that it's sure are part of the MicroCloud in case the user messes with the other services) so I think disabling lz4 should be fine. 

Would we be able to turn it on if for some reason MicroCloud's database gets really complex and we want the extra compression?

There was also a performance benefit to turning it off, right?

**Comment by simondeziel** [comment]
> Well MicroCloud's database is very basic (it's only a record of cluster members that it's sure are part of the MicroCloud in case the user messes with the other services) so I think disabling lz4 should be fine.

Agreed, done.

> There was also a performance benefit to turning it off, right?

So I've heard.

**Review by masnax — APPROVED** [review]

**Comment by simondeziel** [comment]
@masnax I reverted to the default config for raft (so with lz4 support enabled) because it felt too risky for migrating existing DBs that would be lz4-compressed.

File: snapcraft.yaml [hunk 1]
```diff
@@ -71,6 +71,7 @@ parts:
     prime:
       - lib/libraft*so*
       - lib/*/libuv.so*
+      - lib/*/liblz4.so*
```
Comment (tomponline) [inline snapcraft.yaml]: @simondeziel why is this different compared to https://github.com/canonical/lxd-pkg-snap/pull/301/files#diff-e68b9afd67bb77dec8eda77796d3e0b784c7011203114996383eed6533be0cefR981 ?

Comment (simondeziel) [reply snapcraft.yaml]: Thanks for catching this, it should have been the same. It is now.

**Review by tomponline — APPROVED** [review]

**Pull request merged**